v2/dtos/responses/device.go

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package responses
+
+import (
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/dtos"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/dtos/common"
+)
+
+// AddDeviceResponse defines the Response Content for POST Device DTOs.
+// This object and its properties correspond to the AddDeviceResponse object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.x#/AddDeviceResponse
+type AddDeviceResponse struct {
```

Comment:
IMO, we should remove `AddDeviceResponse` , `UpdateDeviceResponse` and just use common.BaseWithIdResponse, Defining them this way adds no value. Yes, swagger will need to be updated to reflect this.